### PR TITLE
Feature/940 resources listview

### PR DESCRIPTION
**Pull request by BeriBoss**

### CODE REVIEW

File: AMW_angular/io/src/app/resources/resources-list/resources-list.component.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <th>Actions</th>
+      </tr>
+      </thead>
+      <tbody>
+        @for (resource of resourceGroupList(); track resource.id) {
+          <tr>
+            <td>{{ resource.name }}</td>
+            <td>{{ resource.defaultRelease ? resource.defaultRelease.release : '' }}</td>
+            <td>
+              <a href="/AMW_web/pages/editResourceView.xhtml?ctx=1&id={{ resource.id }}">
```
Comment (BeriBoss) [inline AMW_angular/io/src/app/resources/resources-list/resources-list.component.html:18]: I have a feeling that the link is not working, but I don't know why? @StephGit

Comment (StephGit) [reply AMW_angular/io/src/app/resources/resources-list/resources-list.component.html:18]: @BeriBoss 
Regarding `ResourceGroup.java > createByResource` there is a defaultResource returned. In the old UI the id of the defaultResource or first entry of resourceGroups is mapped. (maybe always the same?)

Tracedown:

ResourceListDataProvider.java -> getResourceGroupsForSelectedResourceType() -> L107
ResourceListController.java -> loadResourceGroupsForResourceType() -> L74
ResourceGroup.java -> createByResource() -> L59

File: AMW_angular/io/src/app/resource/resource.service.ts [hunk 2]
```diff
@@ -1,24 +1,40 @@
...
   name: string;
 }
 
 @Injectable({ providedIn: 'root' })
 export class ResourceService extends BaseService {
+  private resourceType$: Subject<ResourceType> = new Subject<ResourceType>();
+
+  private resourceGroupListForType: Observable<Resource[]> = this.resourceType$.pipe(
+    switchMap((resourceType: ResourceType) => this.getGroupsForType(resourceType)),
+    startWith(null),
```
Comment (StephGit) [inline AMW_angular/io/src/app/resource/resource.service.ts:24]: this operateor should come first to work correctly

File: AMW_angular/io/src/app/resources/resources-page.component.html [hunk 3]
```diff
@@ -2,30 +2,38 @@
...
-        @if (resourceType.hasChildren) {
-        <span>{{ expandedResourceTypeId === resourceType.id ? '-' : '+' }}</span>
+    <div class="row">
+      <div class="col">
+        @if (permissions().canViewResourceTypes) {
+          <nav class="nav flex-column">
+            @for (resourceType of predefinedResourceTypes(); track resourceType.name) {
+              <a (click)="toggleChildrenAndOrLoadResourcesList(resourceType)">{{ resourceType.name }}</a>
+            }
+          </nav>
```
Comment (StephGit) [inline AMW_angular/io/src/app/resources/resources-page.component.html:12]: maybe a little space between predefined and rootResources? dont now

**Review by StephGit — APPROVED** [review]
Beside of the minimal stuff its approved.
If you don't want to resolve the link here, maybe we could create a new story for this

File: AMW_angular/io/src/app/resources/resources-page.component.html (hunk 3 above)
Comment (BeriBoss) [reply AMW_angular/io/src/app/resources/resources-page.component.html:12]: What do you think about that? I have improved the user interface of the resource pages (especially resource types) in an other PR: https://github.com/liimaorg/liima/pull/803. If it's ok, I will make this adjustment there.

image

**Pull request merged**